src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class to save tasks to and load tasks from file on local machine.
+ */
+public class Storage {
+
+    protected String filePath;
+
+    /**
+     * Constructor for Storage class.
+     * Initializes a Storage object that contains filePath
+     * @param filePath file path to file where user wants his task list saved and loaded.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+
+    }
+
+    /**
+     * Saves tasks to file locally.
+     * @param list list of Task objects.
+     * @throws IOException if file path cannot be found
+     */
+    public void saveTasks(ArrayList<Task> list) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        String fileContent = "";
+        for (Task task: list) {
+            int num;
+            String description = task.getDescription();
+            char type = task.getType();
+            if (!task.getDoneStatus()) {
+                num = 0;
+            }
+            else {
+                num = 1;
+            }
+            if (type == 'T') {
+                fileContent += type + " ; " + num + " ; " + description + "\n";
+            }
+            else if (type == 'D') {
+                Deadline deadline = (Deadline) task;
+                fileContent += type + " ; " + num + " ; " + description +  " ; " + deadline.getBy() + "\n";
+            }
+            else { //type 'E'
+                Event event = (Event) task;
+                fileContent += type + " ; " + num + " ; " + description +  " ; " + event.getAt() + "\n";
+            }
+        }
+        fw.write(fileContent);
+        fw.close();
+    }
+
+    /**
+     * Loads tasks from file locally
+     * @return an ArrayList of Task objects
+     */
+    public ArrayList<Task> loadTasks() {
+        ArrayList<Task> list = new ArrayList<>();
```

Comment:
Would "tasks" be more efficient in telling the reader what the content of list while reading the code?